admin/data/sql/migrations/147.do.create-check-table-type.sql

```diff
@@ -0,0 +1,10 @@
+-- Method 2 - use table-valued parameters
+-- create a table type
+CREATE TYPE [mtc_admin].[checkTableType] AS TABLE (
```

Comment:
@jon-shipley shouldn't `pupil_id`, `checkForm_id `, `checkWindow_id `and `school_id` be `not null`?
Also `isLiveCheck` should require a default?

Reply:
@GeorgeChatzigiannis they could be not null.  As it stands though if one of them is null, one of the inserts will fail and the transaction will roll back.

Reply:
@jon-shipley currently they are nullable though? This means the transaction will succeed for any value that is null. Am I missing something?